### PR TITLE
Document console and file watchers

### PR DESCRIPTION
**Pull request by cmontella**

### CODE REVIEW

File: v0.3/handbook/libraries/stdlib.md [hunk 1]
```diff
@@ -718,7 +718,7 @@ commit
       <ul>
         <li><strong>element</strong> - (optional) - the element which has focus during key-up or key-down</li>
         <li><strong>key-code</strong> - the key code of the key to be monitored</li>
-        <li><strong>key</strong> - (optional) - the key to be monitored; an alternative to key-code for a premade list of keys:
+        <li><strong>key</strong> - (optional) - the key to be monitored; an alternative to key-code for a pre-made list of keys:
```
Comment (ibdthor) [inline v0.3/handbook/libraries/stdlib.md]: "premade" is a word, doesn't need a hyphen

File: v0.3/handbook/libraries/stdlib.md [hunk 2]
```diff
@@ -1456,4 +1453,180 @@ bind
   </tr>
 </table>
 
+## File
+
+Watchers that access the filesystem. These watchers only work in headless mode.
```
Comment (ibdthor) [inline v0.3/handbook/libraries/stdlib.md]: "Watchers" -> "Libraries"

File: v0.3/handbook/libraries/stdlib.md (hunk 1 above)
Comment (ibdthor) [inline v0.3/handbook/libraries/stdlib.md]: "premade" is already a word, doesn't need a hyphen

File: v0.3/handbook/libraries/stdlib.md [hunk 3]
```diff
@@ -1456,4 +1453,180 @@ bind
...
+
+  <tr>
+    <td>
+      <ul>
+        <li><strong>text</strong> - The text to write to the console. Text will also be written to stdout.</li>
+      </ul>
+    </td>
+    <td>
+      <code>
+Write a message to the console.
```
Comment (ibdthor) [inline v0.3/handbook/libraries/stdlib.md]: "// ..."

File: v0.3/handbook/libraries/stdlib.md [hunk 4]
```diff
@@ -1456,4 +1453,180 @@ bind
...
+
+  <tr>
+    <td>
+      <ul>
+        <li><strong>text</strong> - The text to write to the console.</li>
+      </ul>
+    </td>
+    <td>
+      <code>
+Write a warning to the console.
```
Comment (ibdthor) [inline v0.3/handbook/libraries/stdlib.md]: "// ..."

File: v0.3/handbook/libraries/stdlib.md [hunk 5]
```diff
@@ -1456,4 +1453,180 @@ bind
...
+
+  <tr>
+    <td>
+      <ul>
+        <li><strong>text</strong> - The text to write to the console. Text will also be written to stderr.</li>
+      </ul>
+    </td>
+    <td>
+      <code>
+Write an error in the console.
```
Comment (ibdthor) [inline v0.3/handbook/libraries/stdlib.md]: "// ..."

File: v0.3/handbook/libraries/stdlib.md [hunk 6]
```diff
@@ -1456,4 +1453,180 @@ bind
...
+      <ul>
+        <li><strong>path</strong> - The path of the file to be read.</li>
+        <li><strong>encoding</strong> - (optional) - The encoding of the file. The default is utf-8.</li>
+        <li><strong>error</strong> - (optional) - If an error is encountered when attempting to read the file, it will be stored here as a #file/error.</li>
+        <li><strong>contents</strong> - The contents of the file. This attribute will have a value once the entire contents of the file are read.</li>
+      </ul>
+    </td>
+    <td>
+      <code>
+Read a file
```
Comment (ibdthor) [inline v0.3/handbook/libraries/stdlib.md:1478]: Most comments in the code are preceded by "// ...", let's just make sure everything looks the same

File: v0.3/handbook/libraries/stdlib.md [hunk 7]
```diff
@@ -1456,4 +1453,180 @@ bind
...
+        <li><strong>path</strong> - The path of the file to be written.</li>
+        <li><strong>encoding</strong> - (optional) - The encoding of the file. The default is utf-8.</li>
+        <li><strong>contents</strong> - The string that will be written to the file</li>
+        <li><strong>error</strong> - (optional) - If an error is encountered when attempting to read the file, it will be stored here as a #file/error.</li>
+        <li><strong>#file/complete</strong> - When the contents are written successfully, the record will be tagged #file/complete.</li>
+      </ul>
+    </td>
+    <td>
+      <code>
+Write a string to a file
```
Comment (ibdthor) [inline v0.3/handbook/libraries/stdlib.md]: "// ..."

File: v0.3/handbook/libraries/stdlib.md [hunk 8]
```diff
@@ -1456,4 +1453,180 @@ bind
...
+  <tr>
+    <td>
+      <ul>
+        <li><strong>code</strong> - The error code</li>
+        <li><strong>message</strong> - The error message</li>
+      </ul>
+    </td>
+    <td>
+      <code>
+Display a message when a file does not exist
```
Comment (ibdthor) [inline v0.3/handbook/libraries/stdlib.md]: "// ..."

File: v0.3/handbook/libraries/stdlib.md (hunk 6 above)
Comment (cmontella) [reply v0.3/handbook/libraries/stdlib.md:1478]: I've got a branch that takes these out, and formats the examples as standard Eve. Going to make a PR for that soon. Putting the // isn't necessary outside a block

File: v0.3/handbook/libraries/stdlib.md (hunk 8 above)
Comment (joshuafcole) [reply v0.3/handbook/libraries/stdlib.md]: Not needed outside of the code ticks.

File: v0.3/handbook/libraries/stdlib.md (hunk 3 above)
Comment (joshuafcole) [reply v0.3/handbook/libraries/stdlib.md]: Not needed outside of the code ticks.

File: v0.3/handbook/libraries/stdlib.md (hunk 4 above)
Comment (joshuafcole) [reply v0.3/handbook/libraries/stdlib.md]: Not needed outside of the code ticks.

File: v0.3/handbook/libraries/stdlib.md (hunk 5 above)
Comment (joshuafcole) [reply v0.3/handbook/libraries/stdlib.md]: Not needed outside of the code ticks.

File: v0.3/handbook/libraries/stdlib.md (hunk 7 above)
Comment (joshuafcole) [reply v0.3/handbook/libraries/stdlib.md]: Not needed outside of the code ticks.

**Review by joshuafcole — APPROVED** [review]
:+1:

**Pull request merged**